content/blog/2019/07/2019-07-30-jenkins-cli.adoc

```diff
@@ -49,25 +64,33 @@ brew install jcli
 It's very simple to install jcli into your Linux OS. Just need to execute a command line at below:
 
 ```
-curl -L https://github.com/linuxsuren/jenkins-cli/releases/latest/download/jcli-linux-amd64.tar.gz|tar xzv
+curl -L https://github.com/jenkins-zh/jenkins-cli/releases/latest/download/jcli-linux-amd64.tar.gz|tar xzv
```

Comment:
BTW snapcraft.io would be great

Reply:
Yes. I've created an issue for this. Try to support snapcraft later. https://github.com/jenkins-zh/jenkins-cli/issues/121